core.d.ts

```diff
@@ -180,7 +183,11 @@ This method can be handy to put in a stream pipeline, but it comes with a price.
 export function fileTypeStream(webStream: AnyWebReadableStream<Uint8Array>, options?: StreamOptions): Promise<AnyWebReadableByteStreamWithFileType>;
 
 export declare class FileTypeParser {
-	detectors: Iterable<Detector>;
+	/**
+	File-type detectors, initialized with a single entry holding the build in detector function.
+	The array is used to iterate over all detectors, as long as the function returns `undefined`.
+	*/
```

Comment:
```suggestion
	/**
	File type detectors.
	
	Initialized with a single entry holding the built-in detector function.
	*/
```

Reply:
> The array is used to iterate over all detectors, as long as the function returns `undefined`.

This is not really relevant here. It can be explained if needed in the detector docs.